modules/jump_subnet/route_table.tf

```diff
@@ -0,0 +1,17 @@
+resource "aws_route_table" "public_route_table" {
+  vpc_id = "${var.vpc_id}"
+
+  route {
+    cidr_block = "0.0.0.0/0"
+    gateway_id = "${var.igw_id}"
```

Comment:
I thought for the public subnet we were going to use a NAT gateway, with the jump subnet using a IGW?